internal/pkg/aws/cloudformation/stack.go

```diff
@@ -99,6 +101,11 @@ func WithDisableRollback() StackOption {
 // StackEvent is an alias the SDK's StackEvent type.
 type StackEvent cloudformation.StackEvent
 
+func (e StackEvent) String() string {
```

Comment:
We can remove this if we don't actually use it! It helped us visualize what information are included in a `StackEvent`, but if it's not used anywhere, we can just keep it on our local work environment.

Reply:
Let me try and let update if needed.

Reply:
Works fine! Removing.